dags/analysis.py

```diff
@@ -0,0 +1,26 @@
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+from datetime import datetime, timedelta
+from utils.gcp import bigquery_etl_query
+
+default_args = {
+    "depends_on_past": False,
+    "start_date": datetime(2020, 3, 29),
+    "owner": "ascholtz@mozilla.com",
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "jmccrosky@mozilla.com",
+        "ascholtz@mozilla.com",
+    ],
+    "email_on_failure": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+with DAG("analysis", default_args=default_args, schedule_interval="0 1 * * *") as dag:
```

Comment:
@jmccrosky As I understand, the data is updated by a databricks notebook on a daily basis. So this task will copy the data from the "internal" to the public table at 1am every day. Does that timing look correct, or is the internal table updated at different times so that the most recent results might be missing?

Reply:
Do you know what time clients daily is generally up to date?  I guess I'd run it shortly after that.

Reply:
I think the ideal flow here would be that we have a task here that waits on clients_daily, then triggers the databricks notebook to run via the databricks API and waits for that job to complete before running the query task. I'm not aware of any prior art for that, though. It's likely out of scope here.

We can check in Airflow when clients_daily tends to be done. Looks like normally 03:00 UTC should be a safe time to start the notebook, although an extra hour of padding would be nice if we can allow it. I don't know how long the notebook generally takes to run.